### DISCUSSION TITLE
NVD Data Feeds - On December 15th, 2023, the NVD plans to retire all legacy data feeds

### DISCUSSION THREAD

**Discussion by RobSHK** [DependencyTrack/dependency-track#3180] [Q&A]
According to [https://nvd.nist.gov/vuln/data-feeds](url), On December 15th, 2023, the NVD plans to retire all legacy data feeds.
Would this affect Dependency Track in any way?

**Comment by nscuro** [comment]
This is currently being worked on here: https://github.com/DependencyTrack/dependency-track/pull/3175

We're hoping to get this out by next week.
